packages/office-ui-fabric-react/src/components/Rating/Rating.styles.ts

```diff
@@ -28,7 +28,7 @@ export function getStyles(props: IRatingStyleProps): IRatingStyles {
     ratingStarBack: [
       'ms-RatingStar-back',
       {
-        color: semanticColors.bodySubtext,
+        color: semanticColors.disabledText,
```

Comment:
disabled should only be used for non-interactable elements

Reply:
I need some semantic color that uses the neutralTertiary color. The only other one is inputBorder. Should I make a new semantic color with a lighter tone than bodySubtext, such as bodyHighlightText?

Reply:
no, the point is that we don't have slightly different shades of gray for everything that's of a given concept. e.g. disabled components shouldn't all be different shades of gray, there should be a single  slot in a single place that you can tweak if you wanted all disabled controls to be a faded red.

use bodySubtext, which is the color we use for metadata, descriptions, captions, etc. note that these kinds of things DO contradict some existing redlines... design is usually okay with it if it's only tweaking it by a shade up or down. if design disagrees, point them to me and i'll discuss it with them

Reply:
For the rating stars, I don't think there is enough contrast between neutralPrimary and neutralSecondary.
![star colors](https://user-images.githubusercontent.com/6496608/34590854-4a1ead46-f16e-11e7-9460-1fcc2e07993f.png)

Reply:
okay yeah that doesn't look good. go ahead and use the palette.neutralTertiary for now for the background